ui/app/scripts/routes.js

```diff
@@ -306,6 +306,24 @@ define(['./app'], function (dashboard) {
         feature: 'tasksEnabled'
       }
     })
+    .state('home.tasks.detailedTask', {
+      url : 'tasks/definitions/{taskName}',
+      templateUrl : taskTemplatesPath + '/detailed-task.html',
+      controller: 'DetailedTaskController',
+      data:{
+        authenticate: true,
+        feature: 'tasksEnabled'
+      }
+    })
+    .state('home.tasks.tabs.createComposedTask', {
+      url: '/createComposedJob',
```

Comment:
It opens up an `Create Composed Task` editor UI tab. I guess it shouldn't be `/createComposedJob`. I have changed it to `/create-composed-task`. It's the same as `/definitions` URL for task definitions tab contents page. The full URL would have `/tasks/` prefix.